### PR TITLE
Changes var to let and allows variables to be varied using var

### PR DESCRIPTION
**Pull request by Ashvin-Ranjan**
So yeah

### CODE REVIEW

File: python/n.py [hunk 1]
```diff
@@ -387,6 +387,9 @@ def eval_command(self, tree):
 			name_type, value = command.children
 			name, type = get_name_type(name_type)
 			self.variables[name] = Variable(type, self.eval_expr(value))
+		elif command.data == "vary":
```
Comment (SheepTester) [inline python/n.py:390]: VARY, epic

**Review by SheepTester — APPROVED** [review]
Approval

**Review by SheepTester — COMMENTED** [review]
Approval

File: python/n.py (hunk 1 above)
Comment (Ashvin-Ranjan) [reply python/n.py:390]: very cool

**Pull request merged**